src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/NewEditCyclePage/PayoutsSection/hooks/usePayoutsTable.tsx

```diff
@@ -170,12 +205,14 @@ export const usePayoutsTable = () => {
 
     const updatedPercentage =
       (_amount / (newDistributionLimit ?? 0)) * ONE_BILLION
-    let adjustedSplits: Split[] = payoutSplits
```

Comment:
AntD fuckery wasn't picking up the updated `payoutSplits` from https://github.com/jbx-protocol/juice-interface/pull/3948/files#diff-759ba9d2f5802c71f399aef65e184645e33ff3f22978f603242538a7da6f6aa7R169 - this change fixed it